pkg/operator/ceph/file/mirror/spec.go

```diff
@@ -76,8 +76,7 @@ func (r *ReconcileFilesystemMirror) makeDeployment(daemonConfig *daemonConfig, f
 			Name:        daemonConfig.ResourceName,
 			Namespace:   fsMirror.Namespace,
 			Annotations: fsMirror.Spec.Annotations,
-			Labels:      controller.CephDaemonAppLabels(AppName, fsMirror.Namespace, config.FilesystemMirrorType, userID, true),
-		},
+			Labels:      controller.CephDaemonAppLabels(AppName, fsMirror.Namespace, config.FilesystemMirrorType, userID, daemonConfig.ResourceName, "CephFilesystem", true)},
```

Comment:
Full CRD name needed here as well? But the short name seems fine too as long as we're consistent.